### DISCUSSION TITLE
Theme Disappeared After Minor Changes?

### DISCUSSION THREAD

**Discussion by sailingthroughemotion** [community/community#34359] [New to GitHub]
_I'll preface this that I just started using GitHub for the first time in my life this week and have no prior programming/coding experience beyond rudimentary HTML and pressing buttons and seeing what happens in RStudio._ 
**Anyway:**
I made a clone of this https://github.com/chrisbobbe/jekyll-theme-prologue in a new repository to use as a template for a portfolio site that I'm trying to make using GitHub Pages. I seem to have followed the instructions word for word and everything was working until at one point I refreshed the URL and it just didn't. I switched the "theme:" to "remote-theme:" like the guy has in his description, and all I was doing was adding some custom text here and there, but now the themed overlay is gone and I have a very raw, basic-looking website. Any experienced designers know where my whoopsie happened?

**Comment by 3kh0** [comment]
Try this, revert it back to `theme:` from `remote-theme:` it worked for me when I cloned your repo!

**Reply by queenofcorgis** [reply]
Thanks for chiming in @3kh0 - @sailingthroughemotion did this help you - if so don't forget to mark it as the answer 😄
